tools/setup/emoji/emoji_setup_utils.py

```diff
@@ -61,18 +62,7 @@ def emoji_names_for_picker(emoji_name_maps: Dict[str, Dict[str, Any]]) -> List[s
 
 
 def get_emoji_code(emoji_dict: Dict[str, Any]) -> str:
-    # Starting from version 4.0.0, `emoji_datasource` package has started to
-    # add an emoji presentation variation selector for certain emojis which
-    # have defined variation sequences. Since in informal environments(like
-    # texting and chat), it is more appropriate for an emoji to have a colorful
-    # display so until emoji characters have a text presentation selector, it
-    # should have a colorful display. Hence we can continue using emoji characters
-    # without appending emoji presentation selector.
-    # (http://unicode.org/reports/tr51/index.html#Presentation_Style)
-    # If `non_qualified` field is present and not None return it otherwise
-    # return `unified` field.
-    emoji_code = emoji_dict.get("non_qualified") or emoji_dict["unified"]
-    return emoji_code.lower()
+    return emoji_to_hex_codepoint(unqualify_emoji(hex_codepoint_to_emoji(emoji_dict["unified"])))
```

Comment:
Now we'll always use the unqualified emoji by unqualifying it ourselves, instead of seeing if it's present on the object. This gives us more consistent behaviour between emojis, and doesn't rely on the upstream package to give us the "non_qualified" name (which is missing for several emoji).